### DISCUSSION TITLE
Aktualizacja AdGuard Home

### DISCUSSION THREAD

**Discussion by corneliusm717** [AdguardTeam/AdGuardHome#3670] [Q&A]
Have a question or an idea? Please search it [on our forum](https://github.com/AdguardTeam/AdGuardHome/discussions) to make sure it was not yet asked. If you cannot find what you had in mind, please [submit it here](https://github.com/AdguardTeam/AdGuardHome/discussions/new).

### Prerequisites

Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**

- [ ] I am running the latest version
- [ ] I checked the documentation and found no answer
- [ ] I checked to make sure that this issue has not already been filed

### Additional Information

Good morning
after installing AdGuard Home on Rasberry Pi 4, I have AdGuard v.0.102.
I wanted to upgrade to v0.106.3 from the github repository but it is not possible. please help, what to do to have the current version.
regards
Krzysztof Olbryk

**Comment by ameshkov** [comment]
Hi, I don't think it's possible to update automatically.

To update manually you should download the AGH binary from the releases page and replace the old one.

**Comment by corneliusm717** [comment]
Hi, thank you for the information. according to the manual it worked and
now is the correct version, v.0.106

regards,

*Krzysztof Olbryk*




pon., 27 wrz 2021 o 11:08 Andrey Meshkov ***@***.***>
napisał(a):
